ironfish/src/network/peers/peer.test.ts

```diff
@@ -5,7 +5,24 @@
 import * as encryption from './encryption'
 
 jest.mock('ws')
-jest.mock('simple-peer')
+jest.mock('node-datachannel', () => {
+  return {
+    PeerConnection: class {
+      onLocalDescription = () => {}
+      onLocalCandidate = () => {}
+      onDataChannel = () => {}
+      createDataChannel = () => ({
+        onOpen: () => {},
+        onError: () => {},
+        onClosed: () => {},
+        onMessage: () => {},
+        close: () => {},
+        isOpen: () => {},
+        sendMessage: () => {},
+      })
+    },
+  }
+})
```

Comment:
this has to be copied into every test now with web rtc?

Reply:
I moved this into the setup file